org.freedesktop.Platform.GL.mesa-stable.json.in

```diff
@@ -20,14 +19,71 @@
     },
     "modules": [
         {
-            "name": "libdrm",
-            "config-opts": [ "--disable-static", "--disable-udev" ],
+            "name": "llvm",
+            "cmake": true,
+            "builddir": true,
+            "build-options" : {
+                "cflags": "-g1",
+                "cxxflags": "-g1"
+            },
+            "cleanup-platform": ["*"],
+            "build-options" : {
+                "arch" : {
+                    "i386" : {
+                        "config-opts" : [
+                            "-DLLVM_TARGETS_TO_BUILD=X86;AMDGPU;NVPTX;BPF"
```

Comment:
This is not right, if this is only needed for mesa we shouldn't need X86 and BPF targets.
Same below.

Reply:
I think X86 might be used by software rendering.